android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/PwoDiscoveryService.java

```diff
@@ -63,6 +69,11 @@
 
   private static final String TAG = "PwoDiscoveryService";
   private static final String NOTIFICATION_GROUP_KEY = "URI_BEACON_NOTIFICATIONS";
+  private static final String SHARED_PREFS_FILE = "pwo_discovery_service_cache";
```

Comment:
Would it be useful to consolidate this preference file with that used in MainActivity?